geometry/area_of_polygon.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+The area of a polygon
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂